sdk/cosmos/azure-cosmos/azure/cosmos/aio/__init__.py

```diff
@@ -19,16 +19,6 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from .container import ContainerProxy
 from .cosmos_client import CosmosClient
-from .database import DatabaseProxy
-from .user import UserProxy
-from .scripts import ScriptsProxy
```

Comment:
All of these proxy objects should remain here - otherwise they're not public, and will never be documented or show up in APIview :)

Reply:
And also added to `__all__`

Reply:
I'll revert these then but I'm confused with how we would want to make it so we only expose the client in that case? I had thought this was the outcome of what we had discussed in the meeting but seems like I'll need some clarification